api/src/main/java/org/openmrs/module/dhisconnector/api/DHISConnectorService.java

```diff
@@ -70,6 +71,8 @@
 	public String uploadMappings(MultipartFile mapping);
 	
 	public String[] exportSelectedMappings(String[] selectedMappings);
+
+	public String[] exportMapping(String mapping) throws IOException;
```

Comment:
how are you invoking this service? please reveal where you finally invoke this method and the entire stack trace

Reply:
Just for testing, I was just using the exiting method and called this method in it. Wait I'll update the PR so you can see. I'll put the stack trace as well.

Reply:
The PR is updated 
And here's the stack trace, it was not shown in the openmrs.log so it was hard to find sorry for the delay
https://pastebin.com/1magaj3a

Reply:
want to know how u are running that method, so push that piece in this same PR

Reply:
please attach a whole stack trace without choosing a section of it

Reply:
Sure, let me check for it on the console, I couldn't find the whole stack on it last time
I copied what was returned to the UI, Sorry for making a mess

Reply:
@k-joseph I was trying to start the application but this time it crashes even before the login
This is the error I'm getting now 
https://pastebin.com/0vzwXi1F